apps/condo/domains/ticket/schema/TicketReportService.js

```diff
@@ -45,7 +45,8 @@ const TicketReportService = new GQLCustomSchema('TicketReportService', {
             schema: 'ticketReportWidgetData(data: TicketReportWidgetInput!): TicketReportWidgetOutput',
             resolver: async (parent, args, context, info, extra) => {
                 const { periodType, offset = 0, userOrganizationId } = args.data
-                const hasAccess = await checkOrganizationPermission(context.authedItem.id, userOrganizationId, 'canManageTickets')
+
+                const hasAccess = await checkUserBelongsToOrganization(context.authedItem.id, userOrganizationId)
```

Comment:
you should move this logic inside the access. You also should rename the access function name